modules/@apostrophecms/schema/lib/addFieldTypes.js

```diff
@@ -681,7 +681,7 @@ module.exports = (self) => {
       // Allow for ranges to go unset
       // `min` here does not imply requirement, it is the minimum value the range UI will represent
       if (
-        !data[field.name] ||
+        typeof destination[field.name] !== 'number' ||
```

Comment:
using the already-laundered `destination.def` (to have the default value from it) and checking if it's a number so 0 works